### DISCUSSION TITLE
Spawn point for town and outpost went missing

### DISCUSSION THREAD

**Discussion by JeffDogV2** [TownyAdvanced/Towny#5215] [Q&A] [answered]
Hi, I got a question, regarding spawn point went missing. My server outpost spawn point all went missing and I want to recover it. All the town outposts still remain claimed and fine but only the spawn point went missing. Is there a way to check the town plot claimed or a way to recover it ?

![Screenshot (1467)](https://user-images.githubusercontent.com/89342601/130354526-c4cce39f-8324-48be-bb75-1e0457d3c8b8.png)

**Comment by LlmDl** [comment, marked as answer]
Haven't had any reports of missing spawn points. I could see it happening if you changed a world name, change the townblocks' world folders but didn't change the spawn points' names  in the town files.

**Reply by JeffDogV2** [reply]
may I know where can I find the folder directory?

**Reply by LlmDl** [reply]
Spawn points and outposts are all stored in the towny\data\towns\TOWNNAME.txt files.

**Reply by JeffDogV2** [reply]
Our server accidentally created a new world, right now we only have the old data of deleted town spawn point.
All the current town and outpost spawn point went missing after the created new world incident, is there a backup folder that we could find ?

**Reply by LlmDl** [reply]
towny\backups\

**Reply by JeffDogV2** [reply]
Hey LlmDl , the issue has been fixed, your help is much appreciated , thank you.
